attractions/file-input/file-input.svelte

```diff
@@ -27,7 +28,7 @@
   export let vertical = false;
   /**
    * The user's selection. If `multiple` is `false`, the value is an actual `File` object, not a one-element `FileList`, as opposed to the native `<input type="file">`.
-   * @type {File | FileList | null}
+   * @type {File | File[] | null}
```

Comment:
Again, are you sure about this? I see from the initial value that this is correct, but in the usage later on?

Reply:
No, I'm not so sure about this one, it just seemed to silence the errors in our code. As I said, not everything is fixed, but this here has a bigger problem than types, imo. It is sometimes a `File[]` (whenever we set it manually), but other times a `FileList` (when the `<input>` is updated), and neither of them is assignable to the other.

Reply:
I suggest we revert this change for now, lest anything gets out of hand, and come back to it in a separate PR